torat_client/rpcapi.go

```diff
@@ -71,9 +73,29 @@ func (a *API) LS(v shared.Void, r *shared.Dir) (err error) {
 	return
 }
 
-func (a *API) Ping(v shared.Void, r *string) error {
-	// TODO implement
-	*r = "Pong"
+func (a *API) Speedtest(v shared.Void, r *shared.Speedtest) error {
+
+	url := "https://api.ipify.org?format=text"
+	resp, _ := http.Get(url)
```

Comment:
Every value you ignore with "_" in this function is an error value, please handle these, otherwise your code might panic on some nil pointer.

Reply:
So I have been ignoring non-critical errors on the client end to avoid as many faults as possible. Basically, I don't want to return the function if there is a failure and since I am on the client end, printing the output is pointless.

Thankfully this is moot since this whole code block is going to get deleted